src/htmlContent/healthdata-notification-dialog.html

```diff
@@ -0,0 +1,16 @@
+<div class="health-data-notification modal">
```

Comment:
@ryanstewart I'd be interested in your thoughts on the desired UX for this.  Currently, it's _much_ more heavy-handed than what Firefox does, and they've been sort of our model for this feature...